src/aznfswatchdog

```diff
@@ -131,13 +131,26 @@ reconcile_conntrack()
 
 vecho "Starting aznfswatchdog..."
 
+# Detect and log distro and bash version
+if [ -f /etc/centos-release ]; then
+    linux_distro=$(cat /etc/centos-release)
+elif [ -f /etc/os-release ]; then
+    linux_distro=$(grep "^PRETTY_NAME=" /etc/os-release | awk -F= '{print $2}' | tr -d '"')
+else
+    linux_distro="Unknown"
+fi
+
+bash_version=$(bash --version | head -n 1)
+vecho "Linux distribution: $linux_distro"
+vecho "Bash version: $bash_version"
+
 # Dump NAT table once on startup in case we have reported conflicts.
 vecho "NAT table:\n$(iptables-save -t nat)"
 conntrack -L > /dev/null
 
 # conntrack timewait timeout higher than the TCP timewait timeout value isn't very valuable.
 conntrack_timeo_timew=$(cat /proc/sys/net/netfilter/nf_conntrack_tcp_timeout_time_wait 2>/dev/null)
-if [ $? -eq 0 -a $conntrack_timeo_timew -gt $AZNFS_TIMEWAIT_TIMEOUT ]; then
+if [ $? -eq 0 ] && [ $conntrack_timeo_timew -gt $AZNFS_TIMEWAIT_TIMEOUT ]; then
```

Comment:
let's do it like this
if [ $? -eq 0 ] && [ -n "$conntrack_timeo_timew" -a $conntrack_timeo_timew -gt $AZNFS_TIMEWAIT_TIMEOUT ]; then

basically there were some older versions of kernel where this particular setting may not be present hence I think that error might be seen.
Best to check if $conntrack_timeo_timew is a valid value and only then we should check it and try to set it